pkg/localmetrics/localmetrics.go

```diff
@@ -89,16 +92,29 @@ var (
 
 // UpdateAWSMetrics updates the total AWS Accounts metric every N hours
 func UpdateAWSMetrics(kubeClient kubeclientpkg.Client, hour int) {
-	awsClient, _ := awsclient.GetAWSClient(kubeClient, awsclient.NewAwsClientInput{
+	metricLogger := log.WithValues("Namespace", "aws-account-operator-operator")
+	metricLogger.Info("Metrics for AWS Operator")
+
+	awsClient, err := awsclient.GetAWSClient(kubeClient, awsclient.NewAwsClientInput{
 		SecretName: account.AwsSecretName,
 		NameSpace:  awsv1alpha1.AccountCrNamespace,
 		AwsRegion:  "us-east-1",
 	})
 
+	if err != nil {
+		metricLogger.Error(err, "Failed to get awsClient")
+		return
+	}
+
 	d := time.Duration(hour) * time.Minute
 	for range time.Tick(d) {
-		accountTotal, _ := account.TotalAwsAccounts(awsClient)
-		MetricTotalAWSAccounts.With(prometheus.Labels{"name": "aws-account-operator"}).Set(float64(accountTotal))
+		accountTotal, err := account.TotalAwsAccounts(awsClient)
+
+		if err != nil {
+			metricLogger.Error(err, "Failed to get account count, not updating metric.")
```

Comment:
What would be better here is if `TotalAwsAccounts() returned the AWS error code and we printed it here eg `fmt.Sprintf(err, "Failed to get total number of AWS accounts: %s", err)`

Reply:
Done